app/javascript/application.ts

```diff
@@ -1,5 +1,5 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
+import * as Turbo from "@hotwired/turbo-rails";
```

Comment:
For some reason I was getting an exception around missing Turbo when starting up the server, so I specifically imported to Turbo. I'm truthfully not sure why this was happening for me as this must be working in production, if you have any thoughts maybe I could fix something else locally to avoid this.